pandas/io/pytables.py

```diff
@@ -1106,6 +1106,46 @@ def groups(self):
                  g._v_name != u('table'))))
         ]
 
+    def walk(self, where="/"):
+        """ Walk the pytables group hierarchy yielding the group name and
```

Comment:
should be a single sumary line, then a multi-line extended summary

Reply:
done